### DISCUSSION TITLE
Welcome to PyNeuraLogic Discussions!

### DISCUSSION THREAD

**Discussion by LukasZahradnik** [LukasZahradnik/PyNeuraLogic#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joaquincabezas** [comment]
Hi, I'm Joaquín Cabezas and I am exploring LRNNs as part of my PhD. I was looking for a framework that treats relations as first-class citizens and provide some interpretation/reasoning. Testing this Python frontend for NeuraLogic was my make-or-break moment, and as you can imagine, the outcome was positive!

Before this, I was trying interpretability methods on GNNs but felt a little bit disappointed. Now I am replicating my prior work on this language and I find it compelling. 

While using the library I am finding some weak spots, and I will try to contribute to the project with updates/patches. I am also happy to help on the issues/discussions here on Github.

Regards,
Joaquín.

**Reply by GustikS** [reply]
Hi Joaquín, welcome! That sounds interesting and fitting, hope you'll find the framework useful. We'll be glad for your contributions!
Best wishes,
Gustav
